package-lock.json

```diff
@@ -1,4855 +1,4855 @@
-            }
-        },
-        "typescript": {
-            "version": "4.5.5",
-            "dev": true
-        },
-        "undici": {
-            "version": "5.14.0",
-            "resolved": "https://registry.npmjs.org/undici/-/undici-5.14.0.tgz",
-            "integrity": "sha512-yJlHYw6yXPPsuOH0x2Ib1Km61vu4hLiRRQoafs+WUgX1vO64vgnxiCEN9dpIrhZyHFsai3F0AEj4P9zy19enEQ==",
-            "requires": {
-                "busboy": "^1.6.0"
-            }
-        },
-        "uri-js": {
-            "version": "4.4.1",
-            "dev": true,
-            "requires": {
-                "punycode": "^2.1.0"
-            }
-        },
-        "uuid": {
-            "version": "3.3.2"
-        },
-        "v8-compile-cache": {
-            "version": "2.3.0",
-            "dev": true
-        },
-        "validator": {
-            "version": "13.7.0"
-        },
-        "vscode-oniguruma": {
-            "version": "1.6.1",
-            "dev": true
-        },
-        "vscode-textmate": {
-            "version": "5.2.0",
-            "dev": true
-        },
-        "which": {
-            "version": "2.0.2",
-            "dev": true,
-            "requires": {
-                "isexe": "^2.0.0"
-            }
-        },
-        "word-wrap": {
-            "version": "1.2.3",
-            "dev": true
-        },
-        "workerpool": {
-            "version": "6.2.0",
-            "dev": true
-        },
-        "wrap-ansi": {
-            "version": "7.0.0",
-            "dev": true,
-            "requires": {
-                "ansi-styles": "^4.0.0",
-                "string-width": "^4.1.0",
-                "strip-ansi": "^6.0.0"
-            }
-        },
-        "wrappy": {
-            "version": "1.0.2",
-            "dev": true
-        },
-        "y18n": {
-            "version": "5.0.8",
-            "dev": true
-        },
-        "yallist": {
-            "version": "4.0.0",
-            "dev": true
-        },
-        "yargs": {
-            "version": "16.2.0",
-            "dev": true,
-            "requires": {
-                "cliui": "^7.0.2",
-                "escalade": "^3.1.1",
-                "get-caller-file": "^2.0.5",
-                "require-directory": "^2.1.1",
-                "string-width": "^4.2.0",
-                "y18n": "^5.0.5",
-                "yargs-parser": "^20.2.2"
-            }
-        },
-        "yargs-parser": {
-            "version": "20.2.4",
-            "dev": true
-        },
-        "yargs-unparser": {
-            "version": "2.0.0",
-            "dev": true,
-            "requires": {
-                "camelcase": "^6.0.0",
-                "decamelize": "^4.0.0",
-                "flat": "^5.0.2",
-                "is-plain-obj": "^2.1.0"
-            }
-        },
-        "yn": {
-            "version": "3.1.1",
-            "dev": true
-        },
-        "yocto-queue": {
-            "version": "0.1.0",
-            "dev": true
-        }
-    }
-}
+{

```

Comment:
Why is there such a big diff in the lock file?

Reply:
Well it's only whitespace changes. I've had these changes show up on my local branch recently every time I run `npm install` but didn't really think about it. Can one of you pull my branch, run `npm install` and see if your npm changes it back? I'm curious if we have competing `npm` formatting going on here. I'm on npm v8.10.0 btw

Reply:
tl;dr

If Hossam runs `npm install` from my branch the line endings stay as-is, so we decided I'm okay to merge this.

---

More details:

Tested with Hossam offline. It turns out my `npm` is converting `LF` line endings to `CRLF` line endings to match what's in the `package.json` file. The entire rest of the durable repo is using `CRLF` (probably because of [this](https://github.com/Azure/azure-functions-durable-js/blob/v2.x/.editorconfig#L7)), so in a way my `npm` is just making `package-lock.json` consistent with the rest of the repo. The other js repos are using `LF` as far as I checked, but I didn't feel like trying to change durable